pandas/core/internals/blocks.py

```diff
@@ -3145,17 +3138,6 @@ def _merge_blocks(blocks, dtype=None, _can_consolidate=True):
     return blocks
 
 
-def _vstack(to_stack, dtype):
```

Comment:
This was only used once, and doesn't need to be defined at all once the shim isn't necessary anymore